styles/main.css

```diff
@@ -0,0 +1,651 @@
+* {
+  box-sizing: border-box;
+}
+
+/* FONTS */
+@font-face {
+  font-family: "chirp";
+  src: url("../fonts/chirp-regular-web.woff") format("woff");
+  font-weight: normal;
+}
+@font-face {
+  font-family: "chirp";
+  src: url("../fonts/chirp-medium-web.woff") format("woff");
+  font-weight: 500;
+}
+
+@font-face {
+  font-family: "chirp";
+  src: url("../fonts/chirp-bold-web.woff") format("woff");
+  font-weight: bold;
+}
+
+@font-face {
+  font-family: "chirp";
+  src: url("../fonts/chirp-heavy-web.woff") format("woff");
+  font-weight: 800;
+}
```

Comment:
I used "chirp" too, however I lost the ability to make text bold above 600. I think this would have been the solution

Reply:
I also had some issues with the font, was kinda off sometimes, took some time to figure it out